cloudigrade/config/settings/base.py

```diff
@@ -30,6 +30,7 @@
 
 # AWS Defaults
 S3_DEFAULT_REGION = env('S3_DEFAULT_REGION', default='us-east-1')
+S3_BUCKET_NAME = env('S3_BUCKET_NAME', default='')
```

Comment:
Can we default this to something like `"{AWS_SQS_QUEUE_NAME_PREFIX}-cloudigrade-s3"`.  Perhaps `AWS_SQS_QUEUE_NAME_PREFIX` could be generalized so to `AWS_USER_NAME_PREFIX`.